### PR TITLE
Initial preparations for OKD ci

### PR DESCRIPTION
**Pull request by rmohr**
Provide a simple container which has everything needed in place to
deploy HCO on OKD CI and do some basic install sanity checks.

### CODE REVIEW

File: tests/test.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+# Install HCO
+curl https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-operator/master/deploy/deploy.sh | bash
```
Comment (rthallisey) [inline tests/test.sh]: Might as well build the script into the container

Comment (rmohr) [reply tests/test.sh]: true

Comment (rmohr) [reply tests/test.sh]: changed.

File: tests/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Test HCO
+
+Provide a simple container which contains `kubectl` and can retrieve and
+install HCO.
+
+## How to run it
+
+```
+docker run --rm -it -v $PWD/kubeconfig:/kubeconfig -e KUBECONFIG=/kubeconfig kubevirt/hco-tests:latest
```
Comment (rthallisey) [inline tests/README.md]: At some point we should publish the image to a registry.

Comment (rmohr) [reply tests/README.md]: yep, also releasing manifests would be helpful on the ghithub release pages which are bound to a release.

**Comment by rthallisey** [comment]
ci test please

**Pull request merged**